osv2/executor.go

```diff
@@ -0,0 +1,40 @@
+package osv2
+
+import (
+	"context"
+
+	"github.com/opensearch-project/opensearch-go/v2"
+
+	"github.com/CrowdStrike/opensearchtools"
+)
+
+// OSv2Executor is an executor for OpenSearch 2.
+type OSv2Executor struct {
```

Comment:
Call this `Executor` since it will be referenced as `osv2.Executor` on the consumer side.  The `OSv2` prefix stutters.

Reply:
Done